src/views/Timesheets/components/SavedShiftsList/index.js

```diff
@@ -0,0 +1,306 @@
+            </Typography>
+          </MenuItem>
+          <MenuItem label="reporting supervisor" value={this.state.reportingSupervisor}>
+            <Typography>
+              {this.state.reportingSupervisor !== null && this.state.reportingSupervisor.name}
+            </Typography>
+          </MenuItem>
+        </Select>
+      </FormControl>
+    );
+
+    let content = <></>;
+    if (this.props.shifts.length === null) {
+      content = (
+        <Card>
+          <CardContent>
+            <GordonLoader />
+          </CardContent>
+        </Card>
+      );
+    } else if (this.props.shifts.length > 0) {
+      content = (
+        <>
+          {confirmationBox}
+        <Card>
+          <CardContent>
+            <CardHeader title={cardTitle} />
+            <Grid
+              className="shift-list"
+              container
+              spacing={2}
+              justify="space-around"
+              alignItems="center"
+              alignContent="center"
+              style={styles.boxShadow}
+            >
+              {header}
+              {shiftsList}
+            </Grid>
+          </CardContent>
+          {cardTitle === "Submitted Shifts" &&
+              <CardContent>
+                <Grid container>
+                  <Grid item xs={12} sm={6}>
+                    <Typography variant="h6">Total hours worked: {totalHoursWorked}</Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={6}>
+                    <Typography variant="h6">Estimated gross pay: ${totalEstimatedPay}</Typography>
+                  </Grid>
+                </Grid>
+              </CardContent>
+          }
+          {cardTitle === "Approved Shifts" &&
+              <CardContent>
+                <Grid container>
+                  <Grid item xs={12} sm={6}>
+                    <Typography variant="h6">Total hours worked: {totalHoursWorked}</Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={6}>
+                    <Typography variant="h6">Estimated gross pay: ${totalEstimatedPay}</Typography>
+                  </Grid>
+                </Grid>
+              </CardContent>
+          }
+          <CardActions>
+            {cardTitle === "Saved Shifts" && <Grid container>
+              <Grid container>
+                <Grid item xs={6}>
+                  {supervisorDropdown}
+                </Grid>
+                <Grid item xs={6}>
+                  <Button
+                    disabled={this.state.selectedSupervisor === null}
+                    variant="contained"
+                    color="primary"
+                    onClick={this.handleSubmitButtonClick}
+                  >
+                    Submit All Shifts
+                </Button>
+                </Grid>
+              </Grid>
+            </Grid>}
+          </CardActions>
+        </Card>
+        </>
+      );
+    }
+    return <>{content}</>;
```

Comment:
Would it work to just return `content`? (with no braces around it)

Reply:
Ah, the weirdly useful quirk of JSX. Because `content` is a variable, we need the braces here to access what's stored in it. In this case, it's just more JSX, but it's displayed differently based on truth conditions, which is why it's in a variable in the first place. If we didn't have the braces, instead of the actual content, the page would display the literal word "content" in it's place.